homeassistant/components/homeassistant_alerts/strings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "issues": {
+    "alert": {
+      "title": "{integration} has an active alert",
```

Comment:
This and description can be improved

Reply:
The title for these will be the title from alerts no? What is this for exactly?

Reply:
Yeah I think we discussed they should be taken from the alerts themselves, as those titles provide context.

(same for the contents, the frontend has added support for Markdown for it).